modes/longitudinal/src/toolbarButtons.js

```diff
@@ -15,16 +15,14 @@ const { windowLevelPresets } = defaults;
  * @param {*} icon
  * @param {*} label
  */
-function _createButton(type, id, icon, label, commands, tooltip, uiType, isActive) {
+function _createButton(type, id, icon, label, commands, tooltip, extraOptions = undefined) {
   return {
     id,
     icon,
     label,
     type,
     commands,
-    tooltip,
-    uiType,
-    isActive,
+    ...extraOptions,
```

Comment:
Is `tooltip` a member of `extraOptions`? If so I think it should be removed from the parameter list on line 18. Furthermore, you will have to double check all the former calls to `_createButton` to be sure they are using the new form correctly. For example, I think the image overlay button is no longer active by default and it should be.

Reply:
It wasn't supposed to get removed, so I added it back in.  It is so common I wanted to leave it.